packages/components/toggle-group/src/toggle-group.scss

```diff
@@ -0,0 +1,59 @@
+:host {
+  --_border-color: var(--sl-color-button-default-outline-idle-border);
+  --_border-radius: var(--sl-border-radius-button-md);
+  --_border-width: var(--sl-border-width-button-outline);
+  --_paddingInline: 1.25rem 1.5rem;
```

Comment:
We are not using tokens for the padding?

Reply:
Fixed